packages/apps/docs/src/components/Search/Search.tsx

```diff
@@ -12,43 +12,28 @@ interface IProps {
 }
 
 export const Search: FC<IProps> = ({ query, hasScroll, limitResults }) => {
-  const [tabName, setTabName] = useState<string | undefined>();
+  const [, setTabName] = useState<string | undefined>();
   const {
+    metadata = [],
     outputStream,
-    handleSubmit: handleSearchSubmit,
+    handleSubmit,
     conversation,
     error,
     isLoading,
-  } = useSearch();
-  const {
-    results: semanticResults,
-    error: semanticError,
-    isLoading: semanticIsLoading,
-    handleSubmit: handleSemanticSubmit,
-  } = useSemanticSearch(limitResults);
+  } = useSearch(limitResults);
 
-  useEffect(() => {
-    if (
-      query !== undefined &&
-      query.trim() !== '' &&
-      tabName !== undefined &&
-      tabName.trim() !== ''
-    ) {
-      if (tabName === 'qa') {
-        // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        handleSearchSubmit(query);
-      }
-      if (tabName === 'docs') {
-        handleSemanticSubmit(query);
-      }
+  const semanticResults = metadata
+    .map((metadata) => ({ ...metadata, filePath: metadata.title })) // TODO delete this line
+    .map(mapMatches);
 
-      analyticsEvent(EVENT_NAMES['click:search'], {
-        query,
-        tabName,
-      });
+  useEffect(() => {
+    if (query !== undefined && query.trim() !== '') {
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      handleSubmit(query);
+      analyticsEvent(EVENT_NAMES['click:search'], { query });
```

Comment:
Note that the `tabName` is gone here. We could restore it, but they now both contain results for the same query so might be confusing? I think it makes sense to have a generic event for the tab clicks (which we likely already have?)